integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -33,18 +41,22 @@ object PluginImplementation {
     BloopKeys.bloopInstall := PluginDefaults.bloopInstall.value
   )
 
+  import Compat._
   val configSettings: Seq[Def.Setting[_]] = {
     val rawSettingsInConfigs = List(
-      BloopKeys.bloopProductDirectories := PluginDefaults.generateBloopProductDirectories.value,
+      BloopKeys.bloopProductDirectories := List(BloopKeys.bloopClassDirectory.value),
+      BloopKeys.bloopManagedResourceDirectories := PluginDefaults.managedResourceDirs.value,
+      BloopKeys.bloopClassDirectory := PluginDefaults.generateBloopProductDirectories.value,
```

Comment:
Maybe rename `generateBloopProductDirectories` to `generateBloopClassDirectory` ?

Reply:
No, because it's named after `productDirectories` in sbt.